### DISCUSSION TITLE
Ignore particular bean names

### DISCUSSION THREAD

**Discussion by IvanVas** [mapstruct/mapstruct#2793] [Q&A]
The autogenerated code we use (OpenAPI model) adds 'remove' methods for Lists.
We'd like to ignore those in the Mapstruct target (both top level and in sub-collections). Is it possible?

If not, what's the plan B? We can ignore those top-level, ugly though it looks. How do we do this for sub-collections in the target?

Thanks!

**Comment by DanielScholzde** [comment]
We have the same issue. We want to ignore methods with a certain pattern from getting mapped.

I tried to use the MappingExclusionProvider, but that helps only if I wanted to exclude a whole class.

**Comment by filiphr** [comment]
The best place to achieve this is to write a custom `AccessorNamingStrategy`, that is the SPI that MapStruct uses to determine whether a method should be treated as a getter, setter, adder, presence checker or it is some other type of method.
